runtime/moonbeam/src/lib.rs

```diff
@@ -624,6 +624,9 @@ impl pallet_author_mapping::Config for Runtime {
 	type AuthorId = NimbusId;
 	type DepositCurrency = Balances;
 	type DepositAmount = DepositAmount;
```

Comment:
I just noticed while testing that this is set equal to 100. Did you mean for it to be 100 * UNIT/GLMR/etc or is it just intended to be 100 of the smallest unit?

Reply:
Yeah, so I did that partly because I'm lazy. But partly also because I'm not sure what the right thing to do is.

Maybe I should use GLMR in the moonbeam runtime and UNITS in the moonbase runtime?

Reply:
Seems reasonable, GLMR/UNIT/MSHD/MOVR. 100 feels too low, but I also don't know if the intention is to add an additional 100 * $UNIT requirement for all staking accounts.

If the change is 100  -> 100 * GLMR, the integration tests `transfer_through_evm_to_stake` and `reward_block_author` will need to be updated.

Reply:
I ended up switching to proper units and adjusting the tests as you suggested. I also made a `with_mappings` function on the `ExtBuilder` to make it more clear which accounts are paying a deposit for a mapping in each test.